fdm-app/app/components/custom/combobox.tsx

```
@@ -22,42 +17,47 @@ import {
 import { cn } from "@/lib/utils"
+import { Check, ChevronsUpDown } from "lucide-react"
 
 type optionType = {
     value: string
     label: string
 }
 
 interface ComboboxProps {
-    options: { value: string, label: string }[]
+    options: { value: string; label: string }[]
     form: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)